.github/workflows/tests_primary.yml

```diff
@@ -146,7 +146,7 @@ jobs:
     - run: npx playwright install --with-deps
     - run: |
         ./utils/docker/build.sh --amd64 focal $PWTEST_DOCKER_BASE_IMAGE
-        npx playwright docker build
+        npx playwright ctr build
```

Comment:
this is not a common abbreviation for container, never seen this before, I'd keep it `container`.

Reply:
sure, `container` it is!